src/segment-loader.js

```diff
@@ -3,18 +3,39 @@
  */
 import Playlist from './playlist';
 import videojs from 'video.js';
-import SourceUpdater from './source-updater';
+import work from 'webwackify';
```

Comment:
I am a little saddened that we ended up putting this in `segment loader` and making it more complicated. :(

Reply:
Ya this is something that we want to cleanup and move to its own module